WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -233,17 +294,94 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         }
     }
 
-    /// Adds the sahre button at the right of the nav bar
+    private func configureNavigationBar() {
+        let rightItems = [
+            UIBarButtonItem.fixedSpace(24),
+            configuredMoreButton(),
+            UIBarButtonItem.fixedSpace(24),
+            configuredShareButton(),
+            UIBarButtonItem.fixedSpace(24),
+            configuredBrowserButton()
+        ]
+
+        navigationItem.leftBarButtonItem = configuredBackButton()
+        navigationItem.rightBarButtonItems = rightItems.compactMap({ $0 })
+    }
+
+    private func configuredBackButton() -> UIBarButtonItem {
```

Comment:
```suggestion
    private func configureBackButton() -> UIBarButtonItem {
```

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/commit/8479f0d1ec8913acb4007319d850a95b9f115828